crates/swc_ecma_transforms_module/src/util.rs

```diff
@@ -506,13 +506,34 @@ impl Scope {
             Expr::Call(CallExpr {
                 span,
                 callee: Callee::Import(_),
-                args,
+                mut args,
                 ..
             }) if !folder.config().ignore_dynamic
                 // TODO: import assertion
                 && args.len() == 1 =>
             {
-                folder.make_dynamic_import(span, args)
+                let expr = match *(args.pop().unwrap().expr) {
+                    Expr::Ident(ident) => match Self::fold_ident(folder, ident) {
+                        Ok(mut expr) => {
+                            if let Expr::Member(member) = &mut expr {
+                                if let Expr::Ident(ident) = member.obj.as_mut() {
+                                    member.obj = Box::new(Expr::Paren(ParenExpr {
+                                        expr: Box::new(Expr::Seq(SeqExpr {
+                                            span,
+                                            exprs: vec![Box::new(ident.take().into())],
```

Comment:
Is `SeqExpr` required?
Seems like it has only one expression, and due to how codegen works, the output will be exact the same as the inner expression.